src/arb/arb.jl

```diff
@@ -2002,6 +2002,64 @@ Return the number of partitions $p(n)$ as an element of $r$.
 """
 numpart(n::Int, r::ArbField) = numpart(fmpz(n), r)
 
+################################################################################
+#
+#  Hypergeometric and related functions
+#
+################################################################################
```

Comment:
Maybe it is a good idea to have more sections within the file, but then I would expect the same for `acb.jl`.